third_party/nvfuser/test/test_multicluster_fusion.cpp

```diff
@@ -111,19 +111,19 @@ TEST_F(NVFuserTest, MultiClusterFusion) {
       "AggregateDag's Traversal inputs --> outputs {\n"
       "  AggregateExpr representing Cluster 0.Inputs={T0_g[ iS0{i0}, iS1{i2}, iS2{i3} ], }. Outputs={T1_l[ iS3{i0}, iS4{i2}, iS5{i3} ], }.\n"
       "  AggregateVal representing Val T1_l[ iS3{i0}, iS4{i2}, iS5{i3} ] on cluster 0\n"
-      "  Send/Receive Val {T1_l[ iS3{i0}, iS4{i2}, iS5{i3} ]} from cluster 0 to cluster 2\n"
-      "  AggregateVal representing Val T1_l[ iS3{i0}, iS4{i2}, iS5{i3} ] on cluster 2\n"
-      "  AggregateExpr representing Cluster 2.Inputs={T1_l[ iS3{i0}, iS4{i2}, iS5{i3} ], }. Outputs={T5_l[ rS12{i2}, iS13{i3} ], }.\n"
-      "  AggregateVal representing Val T5_l[ rS12{i2}, iS13{i3} ] on cluster 2\n"
-      "  Send/Receive Val {T5_l[ rS12{i2}, iS13{i3} ]} from cluster 2 to cluster 3\n"
-      "  AggregateVal representing Val T5_l[ rS12{i2}, iS13{i3} ] on cluster 3\n"
       "  Send/Receive Val {T1_l[ iS3{i0}, iS4{i2}, iS5{i3} ]} from cluster 0 to cluster 1\n"
       "  AggregateVal representing Val T1_l[ iS3{i0}, iS4{i2}, iS5{i3} ] on cluster 1\n"
       "  AggregateExpr representing Cluster 1.Inputs={T1_l[ iS3{i0}, iS4{i2}, iS5{i3} ], }. Outputs={T3_l[ rS8{i2}, iS9{i3} ], }.\n"
       "  AggregateVal representing Val T3_l[ rS8{i2}, iS9{i3} ] on cluster 1\n"
       "  Send/Receive Val {T3_l[ rS8{i2}, iS9{i3} ]} from cluster 1 to cluster 3\n"
       "  AggregateVal representing Val T3_l[ rS8{i2}, iS9{i3} ] on cluster 3\n"
-      "  AggregateExpr representing Cluster 3.Inputs={T5_l[ rS12{i2}, iS13{i3} ], T3_l[ rS8{i2}, iS9{i3} ], }. Outputs={T6_g[ iS14{i3} ], }.\n"
+      "  Send/Receive Val {T1_l[ iS3{i0}, iS4{i2}, iS5{i3} ]} from cluster 0 to cluster 2\n"
+      "  AggregateVal representing Val T1_l[ iS3{i0}, iS4{i2}, iS5{i3} ] on cluster 2\n"
+      "  AggregateExpr representing Cluster 2.Inputs={T1_l[ iS3{i0}, iS4{i2}, iS5{i3} ], }. Outputs={T5_l[ rS12{i2}, iS13{i3} ], }.\n"
+      "  AggregateVal representing Val T5_l[ rS12{i2}, iS13{i3} ] on cluster 2\n"
+      "  Send/Receive Val {T5_l[ rS12{i2}, iS13{i3} ]} from cluster 2 to cluster 3\n"
+      "  AggregateVal representing Val T5_l[ rS12{i2}, iS13{i3} ] on cluster 3\n"
+      "  AggregateExpr representing Cluster 3.Inputs={T3_l[ rS8{i2}, iS9{i3} ], T5_l[ rS12{i2}, iS13{i3} ], }. Outputs={T6_g[ iS14{i3} ], }.\n"
       "}\n"
```

Comment:
I don't know why this changed, and I don't know if this change is OK, it seems it is just some order change.

Reply:
@samnordmann will look into it. I think it's fine to change it for now. 

@zasdfgbnm, please update the test if necessary.